src/coreclr/nativeaot/Runtime/CMakeLists.txt

```diff
@@ -229,6 +229,7 @@ if(WIN32)
   add_definitions(-DFEATURE_ETW)
   add_definitions(-DFEATURE_EVENT_TRACE)
   add_definitions(-DFEATURE_SUSPEND_REDIRECTION)
+  add_definitions(-DFEATURE_SUSPEND_APC2)
```

Comment:
The name of the feature flag in CoreCLR is `FEATURE_SPECIAL_USER_MODE_APC`. Can we use the same name in both places?